api/src/main/java/io/smallrye/stork/api/ServiceDiscovery.java

```diff
@@ -22,9 +21,8 @@
      * Optional initialization.
      * This method will be invoked after all service discoveries and load balancers are registered in Stork
      *
-     * @param services a map of all defined services by name, helpful with creating service discovery implementations
-     *        that combine results from multiple services
+     * @param stork the stork instance managing the service.
      */
-    default void initialize(Map<String, Service> services) {
+    default void initialize(Stork stork) {
```

Comment:
was this change required?

Reply:
Actually, yes. By doing that, service discovery providers can register new services after the initialization.